math/program/src/processor.rs

```diff
@@ -0,0 +1,31 @@
+//! Program state processor
+
+use {
+    crate::{approximations::sqrt, instruction::MathInstruction, precise_number::PreciseNumber},
+    borsh::BorshDeserialize,
+    solana_program::{account_info::AccountInfo, entrypoint::ProgramResult, msg, pubkey::Pubkey},
+};
+
+/// Instruction processor
+pub fn process_instruction(
```

Comment:
Is this intended to be used by other programs are is it just for testing? If just for testing, maybe we can `mv math/program libraries/math` and just include the entrypoint for testing instruction counts

Reply:
Done in fe78b0a